relationships/synthesis/EBPF-CLIENT-to-EBPF_REDIS.yml

```diff
@@ -0,0 +1,24 @@
+relationships:
+  - name: clientConnectsToRedisServer
+    version: "1"
+    origins:
+      - EBPF
+    conditions:
+      - attribute: eventType
+        anyOf: [ "Metric" ]
+      - attribute: redis.req_cmd
+        present: true
+    relationship:
+      expires: P75M
+      relationshipType: CONNECTS_TO
+      source:
+        extractGuid:
+          attribute: entity.guid
+      target:
+        lookupGuid:
```

Comment:
Probably it's easier to start with `buildGuid`

Reply:
Switched to `buildGuid`